package.json

```diff
@@ -222,15 +224,14 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "22.4.2",
     "json-loader": "0.5.7",
-    "node-sass": "4.13.1",
```

Comment:
`node-sass` is another deprecated project incompatible with electron 19 and node 16